Description: fix format security warning to
 fix format security warning to avoid build failure with -Werror=format-security
Author: Peter Green <plugwash@p10link.net>
Bug-Debian: http://bugs.debian.org/652845
Origin: vendor

--- agave-0.4.7.orig/src/dialogs/gcs-about-window.cc
+++ agave-0.4.7/src/dialogs/gcs-about-window.cc
@@ -105,7 +105,7 @@ namespace gcs
                 catch (const Glib::Error& e)
                 {
                     Glib::ustring message = "Couldn't load Agave logo image from theme. " + e.what();
-                    g_warning(message.c_str());
+                    g_warning("%s",message.c_str());
                 }
             }
         }
